README.md

```diff
@@ -1,25 +1,64 @@
 ![build](https://github.com/nmeylan/rust-ro/actions/workflows/rust.yml/badge.svg)
-# What is rust-ro?
-rust-ro is a Ragnarok mmo server implementation written in rust. While it is a from scratch implemention it is heavily inspired by [herculesWS](https://github.com/HerculesWS/Hercules) and [rathena](https://github.com/rathena/rathena), for example this implementation support same scripting language for NPC meaning that existing scripts should work on this implementation and use the same database structure than rathena and some naming convention were kept.
 
-## Ambition
+![Rust-RO logo](.github/images/logo.png)
```

Comment:
I'm not sure i want to add a logo right now, if you can remove it
Nice artwork by the way

Reply:
Done!